Resources/Prototypes/Corvax/Entities/Markers/Spawners/jobs.yml

```
@@ -0,0 +1,39 @@
+  components:
+  - type: SpawnPoint
+    job_id: smuggler
+
+- type: entity
+  id: SpawnPointMedSyn
+  parent: SpawnPointPassenger
+  name: MedSyn
+  components:
+  - type: SpawnPoint
+    job_id: MedSyn
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)